Source/GUI/CraftBrowser.cs

```diff
@@ -158,6 +158,11 @@ public static ELCraftBrowser Spawn (ELCraftType type, string profile, SelectFile
 			cb.showMergeOption = showMergeOption;
 			cb.OnBrowseCancelled = onCancel;
 			cb.OnFileSelected = onFileSelected;
```

Comment:
This is probably no longer needed.

Reply:
Ah, should have been mrore clear: OnFileSelected seems to be redundant now, but probably does need testing.